pom.xml

```diff
@@ -541,12 +541,12 @@ Additionally it includes CPD, the copy-paste-detector. CPD finds duplicated code
                     <dependency>
                         <groupId>net.sourceforge.pmd</groupId>
                         <artifactId>pmd-core</artifactId>
-                        <version>5.8.1</version>
+                        <version>6.0.1</version>
```

Comment:
We could remove these dependencies, since 6.0.1 is the default, which is shipped by m-pmd-p 3.9.0.
However, I'd say we leave it, so that we easily can update the underlying PMD version, whenever we release a new one.